[2022-03-18 21:26:27,166] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_b 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 21:26:27,226] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_b 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 21:26:27,226] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 21:26:27,227] {taskinstance.py:1068} INFO - Starting attempt 5 of 5
[2022-03-18 21:26:27,229] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 21:26:27,254] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_tasks.training_model_b> on 2022-03-17T00:00:00+00:00
[2022-03-18 21:26:27,263] {standard_task_runner.py:52} INFO - Started process 9574 to run task
[2022-03-18 21:26:27,270] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'processing_tasks.training_model_b', '2022-03-17T00:00:00+00:00', '--job-id', '52', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpnlmio0gy', '--error-file', '/tmp/tmp8rr9fgj3']
[2022-03-18 21:26:27,273] {standard_task_runner.py:77} INFO - Job 52: Subtask processing_tasks.training_model_b
[2022-03-18 21:26:27,394] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_dag.processing_tasks.training_model_b 2022-03-17T00:00:00+00:00 [running]> on host pc
[2022-03-18 21:26:27,536] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.training_model_b
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T00:00:00+00:00
[2022-03-18 21:26:27,536] {logging_mixin.py:104} INFO - model's accuracy: 1.4727489294947271
[2022-03-18 21:26:27,586] {python.py:151} INFO - Done. Returned value was: None
[2022-03-18 21:26:27,619] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=processing_tasks.training_model_b, execution_date=20220317T000000, start_date=20220318T155627, end_date=20220318T155627
[2022-03-18 21:26:27,697] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-18 21:26:27,722] {local_task_job.py:151} INFO - Task exited with return code 0
